### PR TITLE
fix : syntax highlights & layout

### PR DESCRIPTION
**Pull request by praveenpuglia**
- changed classes & ids into code
- got rid of horizontal scrollbar on page, with a little hacky way though.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,14 +40,14 @@
 <section id="quick-look">
 	<h1>A quick look</h1>
 
-	<p>Get the <code>&lt;button></code> element with the class 'continue' and change its content to 'Next Step...' <a href="http://jquery.com">[example credit]</a></p>
+	<p>Get the <code>&lt;button&gt;</code> element with the class <code>continue</code> and change its content to 'Next Step...' <a href="http://jquery.com">[example credit]</a></p>
```
Comment (LeaVerou) [inline index.html]: `>` actually don't need escaping, only `<` do.

File: style/style.css [hunk 2]
```diff
@@ -383,7 +383,7 @@ body > footer {
 		top: 0;
 		bottom: 0;
 		z-index: -1;
-		width: 50vw;
+		width: 49.6vw;
```
Comment (LeaVerou) [inline style/style.css]: Yeah, that is kinda hacky. I'd rather figure out *why* the scrollbar is appearing first…

**Comment by LeaVerou** [comment]
Thanks! See comments.

File: style/style.css (hunk 2 above)
Comment (praveenpuglia) [inline style/style.css]: `#quick-look pre::after` this acquires 50vw on both sides and it's container and `#quick-look pre.vanilla` & `#quick-look pre.bliss` both have border on their right and left respectively. I think that adds up to the width being more than 100vw? There is something else too but I couldn't find it.

File: index.html (hunk 1 above)
Comment (praveenpuglia) [inline index.html]: didn't know that. sorry!

File: style/style.css (hunk 2 above)
Comment (LeaVerou) [inline style/style.css]: If that’s the case, wouldn't `box-sizing: border-box` fix it?

Comment (praveenpuglia) [inline style/style.css]: I would think so but here's what I see.
Total window size - 1366px
Each `:after` element taking 683px themselves. 
We still have the border left out.

However what's strange is that even after removing the margin and border on parent element, it still occupies some space beyond the window width! 

image

**Comment by LeaVerou** [comment]
Wait a second, why is that a problem? Does it cause horizontal scrolling in your browser?

**Comment by praveenpuglia** [comment]
yeah! it does! i tried it on multiple screens.

**Comment by LeaVerou** [comment]
Odd, on OSX it doesn't. What OS is this? Can't tell from the screenshot.

**Comment by praveenpuglia** [comment]
I am on Ubuntu 14.04 - Chrome. Same on Firefox. See a horizontal scrollbar.

**Comment by praveenpuglia** [comment]
Confirm the same for Windows - 7, Chrome, Firefox.

**Comment by kurtextrem** [comment]
chrome v48 Windows 7 also has a horizontal scrollbar.

**Comment by LeaVerou** [comment]
Closing this, please send a new PR with the `calc()` fix discussed in the chat?

**Comment by praveenpuglia** [comment]
sure!

**Pull request closed without merging**